runtime/doc/news.txt

```diff
@@ -237,6 +237,12 @@ The following deprecated functions or APIs were removed.
 • |LanguageTree:parse()| no longer returns changed regions. Please use the
   `on_changedtree` callbacks instead.
 
+• `vim.highlight.create()`, `vim.highlight.link()` were removed, use |nvim_set_hl()| instead
+
+• `vim.treesitter.show_tree()` was removed in favor of |vim.treesitter.inspect_tree()|
```

Comment:
no need to mention this, since it was not a thing in 0.8 -- from the point of view of this release news, `inspect_tree` is just a new addition

Reply:
whoops, sorry, GH web interface lied to me -- this should have only referred to the last item; `vim.highlight` functions are still relevant.

Reply:
Is this correct?

![image](https://user-images.githubusercontent.com/56817415/224487271-ff7e0938-19b0-4d8a-a16f-122355c00925.png)

Reply:
Oh I forgot the dot at the end of the sentence

Reply:
Should be good now 👍🏻 

![image](https://user-images.githubusercontent.com/56817415/224487450-29f678c9-c24f-43db-8e7c-93508e2bef11.png)